### ISSUE TITLE
Demo doesn't work

### ISSUE THREAD

**Issue by ayydevara** [SnappCar/ng2-address#4]

**Comment by ben-crowhurst** [comment]
+1 Demo is broken and `npm install` fails also.  This project appears to be dead.

**Comment by tomasklingen** [comment]
Yes, I'm sorry. The project is currently in hibernation. There are no concrete plans to pick it up again.
